### ISSUE TITLE
RBR20 doesn't appear to work???

### ISSUE THREAD

**Issue by 86lava** [bkerler/netgear_telnet#4]
Running Python 3.9.2 on Windows 10.

1st question: root@RBR20:/# brctl showmacs br0  #results in 9 unique mac addresses - ports 1-9 

port no	mac addr	is local?	ageing timer
1	78:d2:94:52:4c:8f	yes	0
2	7e:d2:94:52:4c:8f	yes	0
3	82:d2:94:52:4c:8f	yes	0
4	86:d2:94:52:4c:8f	yes	0
5	78:d2:94:52:4c:91	yes	0
6	7e:d2:94:52:4c:91	yes	0
7	82:d2:94:52:4c:91	yes	0
8	78:d2:94:52:4c:92	yes	0
9	7e:d2:94:52:4c:92	yes	0

I have tried the following command using each of the above nine mac addresses.  Each time I get the same response.  Am testing this on V2.5.1.16 so that I view the debug.htm to see if telnet has been enabled.

C:\Users\Owner\Desktop>py telnet-enable2.py 10.0.0.1 7E:D2:94:52:4C:8F admin <mypassword>
Netgear LBR20 Telnet enabler V2 LBR20(c) B.Kerler 2021
Done sending pw data to 10.0.0.1:23
Done sending hashed_pw data to 10.0.0.1:23

2nd question: Is this the expected response from the script?

Thanks for your efforts on behalf of the community.  Also pretty much a NOOB here as well.

**Comment by bkerler** [comment]
you need to give the right pw, otherwise it won't work. Example : sudo python3 ./telnet-enable2.py 192.168.1.1 A0:40:A0:63:A6:20 admin <mypassword>

**Comment by SwissAdi** [comment]
Hello sorry to re-open the thread. :) I tried it too on my RBR20 but doesn't seems to works. 

I ran : 

`sudo python3 ./telnet-enable.py 192.168.0.2 xx:xx:xx:xx:xx:xx admin 'password'`

But doesn't seems to work has I'm still not able to telnet to it. also the only output I got is :

`Netgear Telnet enabler V3.1 (c) B.Kerler 2021-2023`

I tried to nap the port and have only the follwwings :

```
adi@adi netgear_telnet-main % nmap 192.168.0.2
Starting Nmap 7.94 ( https://nmap.org ) at 2023-12-31 07:22 CET
Nmap scan report for 192.168.0.2
Host is up (0.0034s latency).
Not shown: 997 closed tcp ports (conn-refused)
PORT    STATE SERVICE
53/tcp  open  domain
80/tcp  open  http
443/tcp open  https
```

For what it worth I'm running all of that on OSX and installed telnet via Brew so should be fine on that side but maybe I'm missing some libraries ? could that be possible ?
